Utilities/Doxygen/DoxygenConfig.cmake

```diff
@@ -161,3 +161,11 @@ set(DOXYGEN_GRAPHICAL_HIERARCHY "NO")
 set(DOXYGEN_DOT_IMAGE_FORMAT "svg")
 set(DOXYGEN_DOT_GRAPH_MAX_NODES "150")
 set(DOXYGEN_DOT_MULTI_TARGETS "YES")
+set(DOXYGEN_DOXYGEN_HTML "${ITK_DOXYGEN_HTML}")
+set(DOXYGEN_DOXYGEN_DOCSET "${ITK_DOXYGEN_DOCSET}")
+set(DOXYGEN_DOXYGEN_CHM "${ITK_DOXYGEN_CHM}")
+set(DOXYGEN_DOXYGEN_QHP "${ITK_DOXYGEN_QHP}")
+set(DOXYGEN_DOXYGEN_ECLIPSEHELP "${ITK_DOXYGEN_ECLIPSEHELP}")
+set(DOXYGEN_DOXYGEN_LATEX "${ITK_DOXYGEN_LATEX}")
+set(DOXYGEN_DOXYGEN_RTF "${ITK_DOXYGEN_RTF}")
+set(DOXYGEN_DOXYGEN_XML "${ITK_DOXYGEN_XML}")
```

Comment:
I think these same lines need to be placed in the wrapping code as well.

./Wrapping/DoxygenConfig.cmake

Reply:
I did not get XML output. Were you getting XML output?

I'm not sure why there are two of these files. It is more difficult to maintain.